CoreFarms.cs

```diff
@@ -338,6 +338,9 @@ public void SevenCirclesWar(int level = 100, int gold = 100000000)
         Core.EquipClass(ClassType.Farm);
         Core.SavedState();
         Core.Logger($"Farming {gold} gold using SCW Method");
+        
+        ToggleBoost(BoostType.Gold);
+        ToggleBoost(BoostType.Experience);
```

Comment:
add if(Bot.Player.Level < 100) above the exp line just so it doesnt waste it

Reply:
I think that line should be added directly into ToggleBoost?